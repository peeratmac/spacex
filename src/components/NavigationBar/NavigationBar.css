* {
  box-sizing: border-box;
}

.nav-link-title {
  color: #34495e;
  padding: 10px 15px;
  text-decoration: none;
  text-transform: uppercase;
  text-align: center;
}

.logo {
  margin-left: 30px;
  font-size: 1.45em;
}

.main-nav {
  margin-top: 5px;
}

.nav-header {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  padding-top: 4.5em;
  padding-bottom: 0.5em;
  border: 1px solid #a2a2a2;
  background-color: #f4f4f4;
  box-shadow: 0px 0px 14px 0px rgba(0, 0, 0, 0.75);
  border-radius: 5px;
}
