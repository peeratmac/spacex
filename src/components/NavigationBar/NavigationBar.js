import React from 'react';
import { NavLink, Link } from 'react-router-dom';
import './NavigationBar.css';

export const NavigationBar = () => {
  return (
    <div>
      <h1>Space X</h1>
      <NavLink exact to='/'>
        Home
      </NavLink>
      <NavLink exact to='/favorites'>
        Favorites
      </NavLink>
      <NavLink exact to='/upcoming-launches'>
        Upcoming Launches
      </NavLink>
      <NavLink exact to='/spacex-history'>
        Space X History
      </NavLink>
      <NavLink exact to='/launch-pads'>
        Launch Pads
      </NavLink>
    </div>
  );
};
