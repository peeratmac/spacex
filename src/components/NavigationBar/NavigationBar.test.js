import React from 'react';
import { shallow } from 'enzyme';
import { NavigationBar } from './NavigationBar';

describe('NAVIGATION BAR', () => {
  it('should match the Navigation Bar Snapshot UI', () => {
    let wrapper = shallow(<NavigationBar />);

    expect(wrapper).toMatchSnapshot();
  });
});
