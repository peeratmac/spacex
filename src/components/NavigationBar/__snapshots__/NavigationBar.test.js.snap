// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NAVIGATION BAR should match the Navigation Bar Snapshot UI 1`] = `ShallowWrapper {}`;
